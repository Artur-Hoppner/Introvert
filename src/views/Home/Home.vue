<template>
  <div class="home">
    <p class="color">It starts here</p>
    <SelectedEvent />
    <Events />
  </div>
</template>

<script>
import SelectedEvent from '../../components/SelectedEvent/SelectedEvent';
import Events from '../../components/Events/Events';
export default {
  name: 'Home',
  components: {
    SelectedEvent,
    Events
  }
};
</script>
<style lang="scss" scoped>
.color {
  font-size: 400%;
  color: red;
}
</style>
