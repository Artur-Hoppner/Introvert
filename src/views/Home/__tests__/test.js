import { shallowMount, createLocalVue } from '@vue/test-utils';
import Vuex from 'vuex';
import Home from '@/views/Home/Home.vue';

const localVue = createLocalVue();

localVue.use(Vuex);

describe('/', () => {
  test('Test if Home is rendered with a snapshot', () => {
    //Arrange
    const wrapper = shallowMount(Home, { localVue });
    //Assert
    expect(wrapper.element).toMatchSnapshot();
  });
});
