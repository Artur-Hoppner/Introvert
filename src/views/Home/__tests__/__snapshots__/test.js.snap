// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`/ Test if Home is rendered 1`] = `
<div
  class="home"
>
  <div
    class="container d-flex"
  >
    <div
      class="row align-center justify-center"
    >
      <div
        class="d-flex col-sm-6 col-12"
      >
        <div
          class="v-input theme--light v-text-field v-text-field--filled v-text-field--enclosed v-select"
        >
          <div
            class="v-input__control"
          >
            <div
              aria-expanded="false"
              aria-haspopup="listbox"
              aria-owns="list-5"
              class="v-input__slot"
              role="button"
              style="background-color: rgba(252, 191, 196, 0.114); border-color: #fcbfc41d;"
            >
              <div
                class="v-select__slot"
              >
                <label
                  class="v-label theme--light"
                  for="input-5"
                  style="left: 0px; position: absolute;"
                >
                  Filter Events
                </label>
                <div
                  class="v-select__selections"
                >
                  <input
                    aria-readonly="false"
                    autocomplete="off"
                    id="input-5"
                    readonly="readonly"
                    type="text"
                  />
                </div>
                <div
                  class="v-input__append-inner"
                >
                  <div
                    class="v-input__icon v-input__icon--append"
                  >
                    <i
                      aria-hidden="true"
                      class="v-icon notranslate mdi mdi-menu-down theme--light"
                    />
                  </div>
                </div>
                <input
                  type="hidden"
                />
              </div>
              <div
                class="v-menu"
              >
                <!---->
              </div>
            </div>
            <div
              class="v-text-field__details"
            >
              <div
                class="v-messages theme--light"
              >
                <transition-group-stub
                  class="v-messages__wrapper"
                  name="message-transition"
                  tag="div"
                />
              </div>
            </div>
          </div>
        </div>
      </div>
       
      <button
        class="reset-button v-btn v-btn--contained theme--light v-size--default light"
        type="button"
      >
        <span
          class="v-btn__content"
        >
          Reset
        </span>
      </button>
    </div>
  </div>
   
  <div
    class="container"
  >
    <div
      class="layout row wrap"
    />
  </div>
</div>
`;
